rs/tests/system_tests.bzl

```diff
@@ -293,6 +295,10 @@ def system_test(
     if uses_guestos_dev_test:
         icos_images["ENV_DEPS__GUESTOS_UPDATE_IMG_TEST"] = _guestos + "update-img-test.tar.zst"
 
+    if uses_guestos_recovery_dev:
+        _guestos_recovery_dev = "//ic-os/guestos/envs/recovery-dev:"
+        icos_images["ENV_DEPS__GUESTOS_RECOVERY_DISK_IMG"] = _guestos_recovery_dev + "disk-img.tar.zst"
```

Comment:
what's the value of `_guestos_recovery_dev` if `uses_guestos_recovery_dev` is `False`?